waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -77,49 +77,249 @@ type WakuRLNRelay* = ref object
+  STATIC_GROUP_KEYS* = @[("e9a4d05b1f539d65c59015a079ee89aabeafbcfc9734342d9559f81601e85417",
```

Comment:
Same re resource folder here

Reply:
sure, please see the tracking issue https://github.com/status-im/nim-waku/issues/946